assets/webview/pages/table/index.js

```diff
@@ -124,13 +126,13 @@ VuePage(function(page) {
       },
       drop() {
         if(this.isEditing)
-          return page.noty.warn('请先保存或撤销修改')
+          return page.noty.warn(page.state.l10n.pleaseSaveOrUndoModify)
         const warning = this.pkNames
           .map(name => `${name} 为 ${this.focusedRecord[name]}`)
           .join(' 且 ')
         page.prompt.warn(
-          '确定删除？',
-          '您正在删除 ' + warning + ' 的记录，删除后不可恢复', 
+          page.state.l10n.deleteCheck,
+          warning + ' ' +page.state.l10n.deleteWarning, 
```

Comment:
- Sure to delete?
- You are deleting (warning) records that are NOT recoverable after deletion